packages/api/db/tables.sql

```diff
@@ -216,4 +216,4 @@ CREATE TABLE IF NOT EXISTS backup
     UNIQUE (upload_id, url)
 );
 
-CREATE INDEX IF NOT EXISTS backup_upload_id_idx ON backup (upload_id);
+CREATE INDEX IF NOT EXISTS backup_upload_id_idx ON backup (upload_id);
```

Comment:
No need for this change?

Reply:
Yeah this was a result of me shuffling files around, will remove